network-api/networkapi/wagtailpages/templates/wagtailpages/blog_index_page.html

```diff
@@ -0,0 +1,15 @@
+{% extends "./index_page.html" %}
+
+{% load wagtailcore_tags wagtailimages_tags i18n %}
+
+{% block featured %}
```

Comment:
the new part, which injected featured posts as long as we're not on a /category or /tag view